.pylintdict

```diff
@@ -304,3 +304,8 @@ yoder
 zoufal
 zsh
 θ
+nlopt
```

Comment:
A minor comment - we used to try to keep this in alphabetical order so it was easy to find if a word was in this custom dict, so changes would be done inline with the existing words rather than appending new ones at the end

Reply:
Reordered the final entries alphabetically.